zone/spells.cpp

```diff
@@ -4544,39 +4544,62 @@ bool Mob::SpellOnTarget(
 	return true;
 }
 
-void Corpse::CastRezz(uint16 spellid, Mob* Caster)
+void Corpse::CastRezz(uint16 spell_id, Mob* Caster)
 {
-	LogSpells("Corpse::CastRezz spellid [{}], Rezzed() is [{}], rezzexp is [{}]", spellid,IsRezzed(),rez_experience);
-
-	if(IsRezzed()){
-		if(Caster && Caster->IsClient())
-			Caster->Message(Chat::Red,"This character has already been resurrected.");
-
+	LogSpells("Corpse::CastRezz spell_id [{}], Rezzed() is [{}], rezzexp is [{}], rez_timer enabled:: [{}]", spell_id,IsRezzed(),rez_experience, corpse_rez_timer.Enabled());
```

Comment:
`Corpse::CastRezz` gets automatically passed to logging, remove

Remove `,` after `[{}]` and keep printouts consistent with variable names